tests/ansible_collections/roles/oracle-linux-specific/tasks/boot_standard_kernel.yml

```diff
@@ -1,3 +1,4 @@
+---
 - name: Install standard kernel
   yum:
     name: kernel
```

Comment:
Inna found an issue, with this task when her machine had kernel-transition package installed, this make this tasks skip to install the RHCK kernel. Should be changed to something like kernel.*

Reply:
We agreed on resolving this in a separate PR.